<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="1">
          <widget class="QDateTimeEdit" name="dtDate"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblSport_2">
           <property name="text">
            <string>Deporte</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbSport">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblRegion_2">
           <property name="text">
            <string>Región</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cmbRegion">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblCompetition_2">
           <property name="text">
            <string>Competición</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="cmbCompetition">
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblPlayer1_2">
           <property name="text">
            <string>Local</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblPlayer2_2">
           <property name="text">
            <string>Visitante</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblPick_2">
           <property name="text">
            <string>Pick</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="txtPick"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lblResult_2">
           <property name="text">
            <string>Resultado</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="cmbResult">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Pendiente</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Acertada</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fallada</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Nula</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medio Acertada</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Medio Fallada</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Retirada</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QCheckBox" name="chkFree">
           <property name="text">
            <string>Apuesta Gratuita</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblDate_2">
           <property name="text">
            <string>Fecha</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="txtPlayer1">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="txtPlayer2">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="lblBookie">
           <property name="text">
            <string>Casa</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cmbBookie"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lblMarket">
           <property name="text">
            <string>Mercado</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="cmbMarket"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblTipster">
           <property name="text">
            <string>Tipster</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="cmbTipster"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblStake">
           <property name="text">
            <string>Stake</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QDoubleSpinBox" name="txtStake">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblOne">
           <property name="text">
            <string>Unidad</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QDoubleSpinBox" name="txtOne">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lblBet">
           <property name="text">
            <string>Apuesta</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QDoubleSpinBox" name="txtBet">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="minimum">
            <double>0.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lblQuota">
           <property name="text">
            <string>Cuota</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QDoubleSpinBox" name="txtQuota">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="decimals">
            <number>4</number>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lblProfit">
           <property name="text">
            <string>Beneficio</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QDoubleSpinBox" name="txtProfit">
           <property name="buttonSymbols">
            <enum>QAbstractSpinBox::NoButtons</enum>
           </property>
           <property name="showGroupSeparator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <double>-99999990000.000000000000000</double>
           </property>
           <property name="maximum">
            <double>999999.989999999990687</double>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="4">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="btnCancel">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Cancelar</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnAccept">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Aceptar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="pnlCombined">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Añadir apuesta</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="pnlDate">
           <item>
            <widget class="QLabel" name="lblDate">
             <property name="text">
              <string>Fecha</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlSport">
           <item>
            <widget class="QLabel" name="lblSport">
             <property name="text">
              <string>Deporte</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlRegion">
           <item>
            <widget class="QLabel" name="lblRegion">
             <property name="text">
              <string>Región</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlCompetition">
           <item>
            <widget class="QLabel" name="lblCompetition">
             <property name="text">
              <string>Competición</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlPlayer1">
           <item>
            <widget class="QLabel" name="lblPlayer1">
             <property name="text">
              <string>Local</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlPlayer2">
           <item>
            <widget class="QLabel" name="lblPlayer2">
             <property name="text">
              <string>Visitante</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlPick">
           <item>
            <widget class="QLabel" name="lblPick">
             <property name="text">
              <string>Pick</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlResult">
           <item>
            <widget class="QLabel" name="lblResult">
             <property name="text">
              <string>Resultado</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="pnlButton">
           <item>
            <widget class="QLabel" name="label_17">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>dtDate</tabstop>
  <tabstop>cmbSport</tabstop>
  <tabstop>cmbRegion</tabstop>
  <tabstop>cmbCompetition</tabstop>
  <tabstop>txtPlayer1</tabstop>
  <tabstop>txtPlayer2</tabstop>
  <tabstop>txtPick</tabstop>
  <tabstop>cmbResult</tabstop>
  <tabstop>cmbBookie</tabstop>
  <tabstop>cmbMarket</tabstop>
  <tabstop>cmbTipster</tabstop>
  <tabstop>txtStake</tabstop>
  <tabstop>txtOne</tabstop>
  <tabstop>txtBet</tabstop>
  <tabstop>txtQuota</tabstop>
  <tabstop>txtProfit</tabstop>
  <tabstop>btnAccept</tabstop>
  <tabstop>chkFree</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
